x-pack/legacy/plugins/maps/public/kibana_services.js

```diff
@@ -8,13 +8,18 @@ import {
   getRequestInspectorStats,
   getResponseInspectorStats,
 } from '../../../../../src/legacy/ui/public/courier';
-export { xpackInfo } from 'plugins/xpack_main/services/xpack_info';
 import { esFilters } from '../../../../../src/plugins/data/public';
-import { npStart } from 'ui/new_platform';
 
 export const SPATIAL_FILTER_TYPE = esFilters.FILTERS.SPATIAL_FILTER;
 export { SearchSource } from '../../../../../src/legacy/ui/public/courier';
-export const indexPatternService = npStart.plugins.data.indexPatterns;
+
+export const license = {};
```

Comment:
export this as a getter function instead. It removes the need for wrapping the id-string in an object,  will hide the implementation details from clients, and named functions make code-navigation easier
e.g.:

```
let licenseId;
export const getLicenseId() {
    return licenseId;
}
```